projects/swych/data/events.js

```diff
@@ -0,0 +1,12 @@
+const EVENTS = [
```

Comment:
these are way too many events, do any of them have material impact on tvl chart? I would strongly recommend skipping those that dont

Reply:
Got you. Thanks for the info. We had no clue that this should be limited. How many events do you recommend displaying?

Reply:
hi @itsmenikolaos ideally only the ones that affect the tvl chart. For example, dont think dashboard launch event has an impact.

I would say, restrict it to two for now :) 

Sorry about refactoring and merging here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/9541 rest of your code is good, I just prefer everything in a single file, easier to read. Can you put up a new PR with 2 events you like to include?